user-org-actor/src/main/java/org/sunbird/user/dao/IUserOSDao.java

```diff
@@ -23,5 +24,13 @@
      */
     Response createUser(Map<String, Object> user) throws BaseException;
 
+    /**
+     * this method will read a user from open saber
+     * @param userId
+     * @return response
+     * @throws BaseException
+     */
+    public Response readUser(JsonNode userId) throws BaseException;
```

Comment:
why requested userId is JsonNode , it should be String

Reply:
This method attribute name incorrect, here we constructed json node with valid input to read method, now attribute name changed